db/migrate/20150129120103_add_default_to_category_ordinal.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultToCategoryOrdinal < ActiveRecord::Migration
+  def change
+    change_column :comfy_cms_categories, :ordinal, :integer, default: 999
```

Comment:
@benlovell Yeah this was a cheap way to ensure new records don't interfere with the order of anything without resorting to callbacks, and I was fairly confident that there wouldn't be more then 999 pages in any one category. I will have another think about a better way to do it though.